frontend/packages/ui/src/toast.tsx

```diff
@@ -195,7 +195,7 @@ export function Toaster() {
           )
           .map((item) => item.key),
       })
-    }, 300)
+    }, 5_000)
```

Comment:
I think this will make the toast interaction feel pretty bad. the duration of toasts will be highly inconsistent and they will transition away at strange times.

I thought you had a nicer solution to fix this involving the items clearing themself?

Reply:
yes I asked for it, and we discussed to refactor it but I did not implement it due to not derail too much from what we decide on HyperTuesday. I can refactor this of course